### ISSUE TITLE
Add configurable cache invalidation settings

### ISSUE THREAD

**Issue by YaroTs** [DylanVann/react-native-fast-image#483]
It seems currently cache invalidation depends on default parameters of underlying cache libs for Android and iOS (Glide and SDWebImage respectively).

Correct me if I'm wrong, but while in Glide cache size is 250 mb and has no expiration time (http://bumptech.github.io/glide/doc/configuration.html#disk-cache), on iOS it's lifetime is limited to one week:  kDefaultCacheMaxCacheAge = 60 * 60 * 24 * 7 in SDImageCacheConfig.h

It would be great if both platforms have the same cache invalidation time (1 week seems to be more reasonable). Do you have any plans to implement extended cache configuration?

**Comment by rewieer** [comment]
I agree. I think caching share a common denominator between both libraries, so things like determining the cache limit and the expiration time could be addressed with the same interface.

Is there any plans about this @DylanVann ? Or would you like a PR ?
